src/containers/Interfaces/NameGenerator.js

```diff
@@ -58,19 +40,34 @@ class NameGeneratorInterface extends Component {
     });
   }
 
-  handleFormSubmit(node, _, form) {
+  handleAddNode = (node, _, form) => {
```

Comment:
It's to indicate an unused parameter provided by SemanticForm. How would you like to see this changed?